controller/src/main/java/org/jboss/as/controller/AttributeDefinition.java

```diff
@@ -108,12 +111,57 @@ protected AttributeDefinition(String name, String xmlName, final ModelNode defau
                                   final ParameterCorrector valueCorrector, final ParameterValidator validator,
                                   boolean validateNull, final String[] alternatives, final String[] requires, AttributeMarshaller attributeMarshaller,
                                   boolean resourceOnly, DeprecationData deprecationData, final AccessConstraintDefinition[] accessConstraints,
-                                  Boolean nilSignificant, AttributeParser parser, final AttributeAccess.Flag... flags) {
+                                  Boolean nilSignificant, AttributeParser parser, final ModelNode undefinedMetricValue, final AttributeAccess.Flag... flags) {
```

Comment:
This is a breaking API change. This multi-param protected method only exists to not break any external code that might have used it or the subclass constructors that use it, so it shouldn't be changed. Any code that wants to use an undefinedMetricValue should use the builder API.

Dropping this param lets you not change List/Map/SimpleAttributeDefinition.